cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,16 @@
+import { generateUser } from '../support/generate';
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const user = generateUser();
+
+    cy.visit('https://conduit.mate.academy/user/register');
+
+    cy.get(':nth-child(1) > .form-control').type(user.username);
+    cy.get(':nth-child(2) > .form-control').type(user.email);
+    cy.get(':nth-child(3) > .form-control').type(user.password);
 
+    cy.get('.btn').click();
+    cy.get(':nth-child(4) > .nav-link').should('contain', user.username.toLowerCase());
```

Comment:
This assertion is checking if the username appears in the navigation bar. However, it would be better to check if the user has been redirected to the appropriate page after registration. You can use 'cy.url().should('include', '/your-expected-url')' for this.